src/sentry/api/event_search.py

```diff
@@ -738,8 +738,7 @@ def visit_paren_group(self, node, children):
                 SearchValue(wrap_free_text(node.text, self.config.wildcard_free_text)),
             )
 
-        children = remove_space(remove_optional_nodes(flatten(children)))
-        children = flatten(children[1])
+        children = remove_space(remove_optional_nodes(flatten(children)))[1:-1]
```

Comment:
this got simpler (and was depending on the buggy behaviour!) -- now that flatten actually works we don't need to double-flatten this (though we do need to remove the braces from the ends!)